README.md

```diff
@@ -65,6 +65,14 @@ The software behind Wikipedia
 * license: CC BY-SA 3.0
 * [website](https://www.mediawiki.org/wiki/MediaWiki) | [repo](https://phabricator.wikimedia.org/diffusion/)
 
+### Oppia
+Oppia is an online leanring tool that enables anyone to create and share interactive activiies.
+In addition the core team also maintains a set of free and effective lessions that are targeted at learners who lack access to educational resources.
```

Comment:
-lessions +lessons

An orthograph checking plugin would help you a lot with finding these typos :)

Reply:
Yeah, it probably is time for that ^^
I think you've recommended one for vim before? What was that again?